src/common/components/Tabs/tabs.scss

```diff
@@ -0,0 +1,38 @@
+.tabsList {
+	display: flex;
+}
+
+.tab {
+	color: #53556A;
+	background-color: #F6FAFD;
+	display: flex;
+	justify-content: center;
+	width: 100%;
+	align-items: center;
+
+	&_active {
+		background-color: #FFFFFF;
+		position: relative;
+
+		&:after {
+			content: '';
+			position: absolute;
+			bottom: 2px;
+			width: calc(100% - 20px);
+			height: 2px;
+			background-color: #E3EAF0;
```

Comment:
check on each step what you use a value witch don't use in another places of your app. if you use color twice, move it into a colors.scss.

About naming in colors.scss - see a https://material.io/design/color/the-color-system.html#tools-for-picking-colors

Reply:
Ok. But this exact color is unique in whole app